content/automate/ManageIQ/Transformation/TransformationThrottler.class/__methods__/utils.rb

```diff
@@ -0,0 +1,125 @@
+module ManageIQ
+  module Automate
+    module Transformation
+      module TransformationThrottler
+        class Utils
+          NAMESPACE = 'Transformation/StateMachines'.freeze
+          CLASS_NAME = 'TransformationThrottler'.freeze
+          DEFAULT_THROTTLER_ELECTION_POLICY = 'eldest_active'.freeze
+          DEFAULT_THROTTLER_TYPE = 'Default'.freeze
+          DEFAULT_THROTTLER_TTL = 3600
+          DEFAULT_TASKS_SCHEDULING_POLICY = 'fifo'.freeze
+          DEFAULT_LIMITS_ADJUSTMENT_POLICY = 'skip'.freeze
+
+          def self.log_and_raise(message, handle = $evm)
```

Comment:
@fdupont-redhat 
@billfitzgerald0120 Has added this 
https://github.com/ManageIQ/manageiq-content/blob/a50ea6c95dfb2e2b51a69720e67f0fae973d1733/content/automate/ManageIQ/System/CommonMethods/Utils.class/__methods__/log_object.rb#L56
which could be used here

Reply:
@mkanoor Good pointer. I'll give it a look for other parts of Transformation namespace, where I have similar needs. This throttling PR is a short term solution, so I don't think I'll do further changes, unless hitting bugs.